frontend/fingertips-frontend/components/views/OneIndicatorOneAreaView/OneIndicatorOneAreaView.test.tsx

```diff
@@ -59,11 +65,43 @@ describe('OneIndicatorOneAreaView', () => {
     }
   );
 
+  it('should call get indicator endpoint and pass indicator metadata if a single indicator is selected', async () => {
```

Comment:
```suggestion
  it('should call get indicator endpoint and pass indicator metadata', async () => {
```

We know a single indicator is always selected here.